File: 0205.py
from typing import List

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        nums[:] = sorted(set(nums))
        return len(nums)


# 測試案例
solution = Solution()
test_input = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4]
result = solution.removeDuplicates(test_input)
print(result)
print(test_input)

File: 0205.js
function removeDuplicates(nums) {
    nums.splice(0, nums.length, ...new Set(nums));
    return nums.length;
}
  
  // 測試案例
  let testInput = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4];
  let result = removeDuplicates(testInput);
  console.log(result);
  console.log(testInput);